### PR TITLE
Upgrade version of commons-compress

### PR DESCRIPTION
**Pull request by Sigmonia**
#### Rationale
Upgrade version of commons-compress

#### Related Pull Requests
* https://github.com/LabKey/platform/pull/5247

#### Changes
* version bump

### CODE REVIEW

**Review by labkey-adam — APPROVED** [review]

File: gradle.properties [hunk 1]
```diff
@@ -128,7 +128,7 @@ commonsCollectionsVersion=3.2.2
 commonsCollections4Version=4.4
 commonsCodecVersion=1.16.0
 # sync with version Tika ships
-commonsCompressVersion=1.24.0
+commonsCompressVersion=1.26.0
```
Comment (labkey-tchad) [inline gradle.properties]: We can probably remove this comment about Tika. This will be at least the second time we've bumped this without touching Tika.

**Review by labkey-tchad — APPROVED** [review]

**Review by labkey-tchad — DISMISSED** [review]
Looks like this breaks the build

**Comment by labkey-tchad** [comment]
Compile error in `IpynbReport.java`:
```
/mnt/teamcity/work/f0afbbc879bedfad/server/modules/platform/api/src/org/labkey/api/reports/report/python/IpynbReport.java:408: error: incompatible types: ArchiveEntry cannot be converted to TarArchiveEntry
                                 tar.putArchiveEntry(entry);
                                                     ^
```
https://teamcity.labkey.org/buildConfiguration/LabKey_2311Release_Premium_VerifyJavaBuild/2911773

**Comment by labkey-adam** [comment]
> Compile error in `IpynbReport.java`:
> 
> ```
> /mnt/teamcity/work/f0afbbc879bedfad/server/modules/platform/api/src/org/labkey/api/reports/report/python/IpynbReport.java:408: error: incompatible types: ArchiveEntry cannot be converted to TarArchiveEntry
>                                  tar.putArchiveEntry(entry);
>                                                      ^
> ```
> 
> https://teamcity.labkey.org/buildConfiguration/LabKey_2311Release_Premium_VerifyJavaBuild/2911773

Yes, I remember having to make a small change when I upgraded to 1.25.0 on develop. See https://github.com/LabKey/platform/pull/5130/files#diff-34a49bce20357fe213173ff5e7c7ac598fe154ea6d0c422137a7ec7749a5842f

**Pull request merged**